src/matrixCells/makeMatrixArgs.js

```diff
@@ -126,8 +165,8 @@ export default function make_matrix_args(regl, store) {
       zoom: zoom_function,
       interp_uni: (ctx, props) => Math.max(0, Math.min(1, props.interp_prop)),
       run_animation: regl.prop("run_animation"),
-      pos_rgb: store.select("cat_viz").mat_colors.pos_rgb,
-      neg_rgb: store.select("cat_viz").mat_colors.neg_rgb,
+      pos_rgb: store.select("cat_viz").mat_colors.pos_rgb.map((x) => x / 255),
```

Comment:
Cool -- I noticed that the colors wouldn't turn out right if I passed in `[1,1,1]` instead of `[255,255,255]` before right? I guess what I'm wondering is why this is needed now and wasn't needed before?

Reply:
In webgl, we should always keep the color value as a float with a range between 0 and 1.
I assume, previously either `regl` or `webgl` take the int value of `255` and convert it back to float automatically.

Reply:
Makes sense.